src_assets/common/assets/web/platform-i18n.js

```diff
@@ -0,0 +1,69 @@
+import {inject} from 'vue'
+
+class PlatformMessageI18n {
+    /**
+     * @param {string} platform
+     */
+    constructor(platform) {
+        this.platform = platform
+    }
+
+    /**
+     * @param {string} key
+     * @return {string} key with platform identifier
+     */
+    getPlatformKey(key) {
+        switch (this.platform) {
+            case 'windows':
+                return key + '_win'
```

Comment:
From my count, there's only 7 keys that use `_win`... perhaps we should just rename these to `_windows`.

Reply:
we can, there's anywhere else we need to update or just rename the `json`s and the `i18n` calls?

Reply:
Yep, just those two locations. `en.json` only, don't worry about the others.